<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainViewClass</class>
 <widget class="QMainWindow" name="MainViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LightFileSync</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="6">
          <widget class="QTableWidget" name="FoldersTable">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed</set>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Left folder</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Right folder</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="RightFolderText">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="DeleteSyncFoldersButton">
           <property name="toolTip">
            <string>Delete</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../MainView.qrc">
             <normaloff>:/MainView/Resources/delete.png</normaloff>:/MainView/Resources/delete.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonFollowStyle</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="6">
          <widget class="QTableWidget" name="SyncTable">
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed</set>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <column>
            <property name="text">
             <string>Left file</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Size</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cat.</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Action</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Right file</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Size</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ok</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="LeftFolderText">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="BrowseFolderLeftButton">
           <property name="text">
            <string>Browse left</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="AddSyncFoldersButton">
           <property name="toolTip">
            <string>Add</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../MainView.qrc">
             <normaloff>:/MainView/Resources/add.png</normaloff>:/MainView/Resources/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonFollowStyle</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="BrowseFolderRightButton">
           <property name="text">
            <string>Browse right</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="Progress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSync"/>
   <addaction name="actionStop"/>
   <addaction name="actionSave"/>
   <addaction name="actionSettings"/>
   <addaction name="actionInfo"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>270</width>
     <height>230</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Workflows</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="AddWorkflowButton">
         <property name="toolTip">
          <string>Add</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../MainView.qrc">
           <normaloff>:/MainView/Resources/add.png</normaloff>:/MainView/Resources/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonFollowStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="DeleteWorkflowButton">
         <property name="toolTip">
          <string>Delete</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../MainView.qrc">
           <normaloff>:/MainView/Resources/delete.png</normaloff>:/MainView/Resources/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonFollowStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="WorkflowText"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="WorkflowItemList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSync">
   <property name="icon">
    <iconset resource="../MainView.qrc">
     <normaloff>:/MainView/Resources/sync.png</normaloff>:/MainView/Resources/sync.png</iconset>
   </property>
   <property name="text">
    <string>Sync</string>
   </property>
   <property name="toolTip">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../MainView.qrc">
     <normaloff>:/MainView/Resources/settings.png</normaloff>:/MainView/Resources/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../MainView.qrc">
     <normaloff>:/MainView/Resources/about.png</normaloff>:/MainView/Resources/about.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Info</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../MainView.qrc">
     <normaloff>:/MainView/Resources/stop.png</normaloff>:/MainView/Resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../MainView.qrc">
     <normaloff>:/MainView/Resources/save.png</normaloff>:/MainView/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../MainView.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mainToolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>MainViewClass</receiver>
   <slot>ToolBarTriggered(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WorkflowItemList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainViewClass</receiver>
   <slot>WorkflowListChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WorkflowItemList</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>WorkflowText</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddWorkflowButton</sender>
   <signal>clicked()</signal>
   <receiver>MainViewClass</receiver>
   <slot>WorkflowListAddClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteWorkflowButton</sender>
   <signal>clicked()</signal>
   <receiver>MainViewClass</receiver>
   <slot>WorkflowListDeleteClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddSyncFoldersButton</sender>
   <signal>clicked()</signal>
   <receiver>MainViewClass</receiver>
   <slot>FoldersTableAddClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteSyncFoldersButton</sender>
   <signal>clicked()</signal>
   <receiver>MainViewClass</receiver>
   <slot>FoldersTableDeleteClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrowseFolderLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>MainViewClass</receiver>
   <slot>FoldersTableLeftBrowseClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrowseFolderRightButton</sender>
   <signal>clicked()</signal>
   <receiver>MainViewClass</receiver>
   <slot>FoldersTableRightBrowseClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>965</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>890</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FoldersTable</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainViewClass</receiver>
   <slot>FoldersTableItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>1017</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ToolBarTriggered(QAction*)</slot>
  <slot>WorkflowListChanged(int)</slot>
  <slot>WorkflowListAddClick()</slot>
  <slot>WorkflowListDeleteClick()</slot>
  <slot>FoldersTableAddClick()</slot>
  <slot>FoldersTableDeleteClick()</slot>
  <slot>FoldersTableLeftBrowseClick()</slot>
  <slot>FoldersTableRightBrowseClick()</slot>
  <slot>FoldersTableItemSelectionChanged()</slot>
 </slots>
</ui>
